### PR TITLE
Decoupled SpectatorWebEndpoints from Stackdriver

### PR DESCRIPTION
**Pull request by ewiseblatt**
@cfieber

### CODE REVIEW

File: kork-stackdriver/src/main/java/com/netflix/spinnaker/config/SpectatorWebEndpointConfig.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import com.netflix.spectator.api.Registry;
+import com.netflix.spectator.controllers.MetricsController;
+
+
+@Configuration
+@ConditionalOnProperty("spectator.webEndpoint.enabled")
+public class SpectatorWebEndpointConfig {
```
Comment (cfieber) [inline kork-stackdriver/src/main/java/com/netflix/spinnaker/config/SpectatorWebEndpointConfig.java]: what is the expectation on how someone initializes this? do they add an `@Import` or `@ComponentScan` to include this configuration?

File: kork-stackdriver/src/main/java/com/netflix/spinnaker/config/SpectatorWebEndpointConfig.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  @Bean
+  MetricsController metricsController(Registry registry) {
```
Comment (ewiseblatt) [inline kork-stackdriver/src/main/java/com/netflix/spinnaker/config/SpectatorWebEndpointConfig.java]: SGTM

File: kork-stackdriver/src/main/java/com/netflix/spinnaker/config/StackdriverConfig.java [hunk 3]
```diff
@@ -17,7 +17,6 @@
 package com.netflix.spinnaker.config;
 
 import com.netflix.spectator.controllers.filter.PrototypeMeasurementFilter;
-import com.netflix.spectator.controllers.MetricsController;
```
Comment (cfieber) [inline kork-stackdriver/src/main/java/com/netflix/spinnaker/config/StackdriverConfig.java]: does this mean that `kork-stackdriver` no longer needs a dependency on `spectatorWebSpring`

**Comment by cfieber** [comment]
just the one comment and LGTM either way that is resolved

File: kork-stackdriver/src/main/java/com/netflix/spinnaker/config/StackdriverConfig.java (hunk 3 above)
Comment (ewiseblatt) [inline kork-stackdriver/src/main/java/com/netflix/spinnaker/config/StackdriverConfig.java]: kork-stackdriver is pulling a model class from it for filtering so I left the dependency there.

**Pull request merged**